### ISSUE TITLE
Plot autocomplete acceptances against chat/command usages

### ISSUE THREAD

**Issue by chenkc805** [sourcegraph/cody#1006]
I think it'd be worth seeing what the usage of autocomplete acceptances vs chat/command usage is at some point. I'm imagining a scatter plot with autocompletes on the x-axis and chat/completions on the y-axis, and each datapoint is a user in a certain month.

Through anecdotal user research, Kalan has observed that internal sourcegraph users use either one or the other, but less commonly both. 

I'd be curious to see if that bears out in the data with our non-SG users. If what Kalan's seeing turns out to be true, could have implications for how we prioritize the client roadmaps IMO.

Context: https://sourcegraph.slack.com/archives/CN4FC7XT4/p1694451243643709

**Comment by chenkc805** [comment]
This is basically already done here: https://docs.google.com/spreadsheets/d/1WyS7rkbavaIuzQom4ifvnQ8qSHOq1zAfG3sEeTe5c1M/edit#gid=2023464953

TLDR There's no clear bimodal trend that users either use one or the other. It seems like people use Cody in all sorts of ways: Lots of autocompletes, lots of chats, and everything in between.

**Comment by csells** [comment]
I see the bimodal chart but I'm not sure what I'm looking at. Can you
provide one more level of detail?

**Comment by vdavid** [comment]
Looks like this is done, after all
